common/types/question.d.ts

```diff
@@ -0,0 +1,14 @@
+interface Question {
```

Comment:
I will suggest using `type Question = {` instead of interface so that we all know we are defining the `Question` type. But if yall prefer using interface, I can follow yours.

Reply:
sure no prob, i'll change to types.